src/controllers/crop_controller.py

```diff
@@ -114,3 +117,42 @@ def add_weed(self, loc):
 
             return weed
 
+    def move_cursor_and_snap_selected_to_cursor(self, x, y, z):
+        # Save the current area type
+        original_area_type = self.context.area.type
+
+        # Switch to 3D viewarea
+        self.context.window.scene = bpy.data.scenes[0]
+        self.context.area.type = "VIEW_3D"
+
+        # Setting the cursor position
+        self.context.scene.cursor.location = (x, y, z)
+
+        # Simulates the operation of adsorbing a selected object to the cursor position.
+        self.simulate_snap_selected_to_cursor()
+
+        # Switch back to the original area
+        self.context.area.type = original_area_type
+
+        print("The objects have been attached to the cursor position.")
```

Comment:
Are these print statements necessary?

Reply:
This is necessary in order to better represent user interactivity, as well as to facilitate better feedback.

Reply:
I can delete that if you want  to.
but I think is better to keep it

Reply:
Sorry could you clarify how the user will interact with it if it's running using the Blender API aka Blender headless

Reply:
I would recommend removing the print statement as it's not signalling a major event in the API.

Reply:
change the print already